### PR TITLE
Feature/SearchNews

### PR DESCRIPTION
**Pull request by asyagraf**

### CODE REVIEW

File: src/SearchService.Bussines/Commands/Search/SearchCommand.cs [hunk 1]
```diff
@@ -134,17 +136,55 @@ private async Task<List<SearchResultInfo>> SearchUsersAsync(string text, List<st
       return users;
     }
 
+    private async Task<List<SearchResultInfo>> SearchNewsAsync(string text, List<string> errors)
+    {
+      List<SearchResultInfo> news = new();
+
+      string errorMessage = "Cannot search news. Please try again later.";
```
Comment (MariyaSin) [inline src/SearchService.Bussines/Commands/Search/SearchCommand.cs:144]: use RequestHandler from brokerSupport v 4

File: src/SearchService/Controllers/SearchController.cs [hunk 2]
```diff
@@ -10,7 +10,7 @@ namespace LT.DigitalOffice.SearchService.Controllers
   [ApiController]
   public class SearchController : ControllerBase
   {
-    [HttpGet]
+    [HttpGet("get")]
```
Comment (MariyaSin) [inline src/SearchService/Controllers/SearchController.cs]: rename to "search"

**Review by borovHokage — APPROVED** [review]

File: src/SearchService.Bussines/Commands/Search/SearchCommand.cs [hunk 3]
```diff
@@ -134,17 +137,50 @@ private async Task<List<SearchResultInfo>> SearchUsersAsync(string text, List<st
...
```
Comment (MariyaSin) [inline src/SearchService.Bussines/Commands/Search/SearchCommand.cs:144]: do you use the string?

File: src/SearchService.Bussines/Commands/Search/SearchCommand.cs [hunk 4]
```diff
@@ -134,17 +137,50 @@ private async Task<List<SearchResultInfo>> SearchUsersAsync(string text, List<st
...
+
+      var response = await RequestHandler.ProcessRequest<ISearchNewsRequest, ISearchResponse>(
```
Comment (MariyaSin) [inline src/SearchService.Bussines/Commands/Search/SearchCommand.cs:146]: do not use var

**Review by MariyaSin — APPROVED** [review]

**Pull request merged**